src/Features/Core/Portable/FeaturesResources.resx

```diff
@@ -1714,4 +1714,7 @@ This version used in: {2}</value>
   <data name="Add_null_checks_for_all_parameters" xml:space="preserve">
     <value>Add null checks for all parameters</value>
   </data>
+  <data name="Add_static_modifier" xml:space="preserve">
+    <value>Add the 'static' modifier</value>
```

Comment:
Maybe `Make member static` or simply `Make static` would be better.

Reply:
Yes, `Make static` sounds better.